### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by wizzy25**

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
+'use strict';
+
+
+var program = require('commander'), fs = require('fs'), nconf = require('nconf'), fileview = require('./index');
```
Comment (hisabimbola) [inline cli.js]: Each declaration should be done on every line.

Makes the code more readable.

I will advice you install [eslint](eslint.org) and lint with at least [airbnb](https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb). Will help improve coding pattern.

File: cli.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+var pkg = JSON.parse(fs.readFileSync(__dirname + '/package.json'));
```
Comment (hisabimbola) [inline cli.js]: you can just do this

```js
const pkg = require('./package.json')
```

File: index.js [hunk 3]
```diff
@@ -1,33 +1,41 @@
 'use strict';
 
-var Dropbox = require('dropbox');
-var fs = require('fs');
+const Dropbox = require('dropbox'), fs = require('fs'), path = require('path');
 
-var token = '';
+function fileview(arg){
+	if(arg.upload) {upload(arg)}
```
Comment (hisabimbola) [inline index.js]: if there is one line statement in the `if` command, you can ignore the braces

```js
if(arg.upload) upload(arg)
```

or for better readability do

```js
if(arg.upload)
  upload(arg)
```

File: index.js [hunk 4]
```diff
@@ -1,33 +1,41 @@
...
+		process.exit(1);
+	}
+}
 
-var dbx = new Dropbox({accessToken: token});
 
 //Function to upload a file to dropbox
-var upload = ()=>{
-	return fs.readFile('./pic.jpg', (err, file)=>{
+var upload = (arg)=>{
```
Comment (hisabimbola) [inline index.js]: Add space before the braces and after the closing brackets.

File: index.js [hunk 5]
```diff
@@ -1,33 +1,41 @@
...
-upload();
+
 //Function to view files in Dropbox
-var show = ()=>{
-	return dbx.filesListFolder({path: '/alles deutsch'})
+var view = (arg)=>{
+	var dbx = new Dropbox({accessToken: arg.token});
+	if (arg.view === '#root' || typeof(arg.view) === 'boolean') {arg.view = ''};
+	return dbx.filesListFolder({path: arg.view})
 	.then(res => {
```
Comment (hisabimbola) [inline index.js:41]: spaced like this is sweet 👌

File: index.js [hunk 6]
```diff
@@ -38,12 +46,23 @@ var show = ()=>{
...
-		console.log("Alles gut: " + JSON.stringify(res, null, 2))})
-	.catch(err=>{console.error("Alles schlecht: %s", JSON.stringify(err, null, 2))});
-};
\ No newline at end of file
+		console.log("Done downloading: " + JSON.stringify(res, null, 2))})
+	.catch(err=>{console.error("Something went wrong: %s", JSON.stringify(err, null, 2))});
+};
+
+//Function to diable access token
+var disable = (arg)=>{
```
Comment (hisabimbola) [inline index.js]: I am not sure this is needed.

What is the use case for this?

**Review by hisabimbola — COMMENTED** [review]
Changes look sweet.

Added few comments about some things that can be improved.

I will really recommend you use eslint and airbnb config. Will help make your code more readable.

Else 🌟

File: index.js (hunk 3 above)
Comment (wizzy25) [reply index.js]: Yeah... initially left it for "readability"

File: index.js (hunk 5 above)
Comment (wizzy25) [reply index.js:41]: :-D

File: index.js (hunk 6 above)
Comment (wizzy25) [reply index.js]: Well, I had to disable my token, and feel someone else might want to... in case it gets compromised 👍

**Pull request merged**